### DISCUSSION TITLE
Missing Elements by Result?

### DISCUSSION THREAD

**Discussion by CaptainInler** [osm-search/Nominatim#2656] [Q&A]
Hi all

I'm new to Nominatim, so maybe I just misunderstand the concept.. 😅
When searching after [Städtle Vaduz](https://www.openstreetmap.org/search?query=St%C3%A4dtle%20Vaduz) there are three Results displayed. However, the "Road" consists out of:

- https://www.openstreetmap.org/way/571067342
- https://www.openstreetmap.org/way/571067337
- https://www.openstreetmap.org/way/28712147
- https://www.openstreetmap.org/way/297699632
- https://www.openstreetmap.org/way/384912013

Why are there not all Elements returned by the Result?

**Comment by lonvia** [comment]
Nominatim hides duplicate identical results. Use `dedupe=0` if you need all matches: https://nominatim.openstreetmap.org/search.php?q=St%C3%A4dtle+Vaduz&format=jsonv2&dedupe=0
